lib/DDG/Spice/GoWatchIt.pm

```diff
@@ -31,7 +31,8 @@ spice wrap_jsonp_callback => 1;
 handle remainder_lc => sub {
 
   return unless $_; # Guard against "no answer"
-  return if ($req->query_lc ~~ @stopwords); # Guard against stop words "apple watch"
+  return if grep {$req->query_lc eq $_} @stopwords;
```

Comment:
@iambibhas it's pretty odd, but grep acts like `map` when you give it a block expression. The difference is that map returns the value of the block, whereas grep returns the current element, if the block evals to true.

So for each item we check if the current item matches our query, and if so we return the item. This of course produces an array as our final value, and if we get an array we want to bail in this case.

Does that make sense?